cmd/vic-init/toolbox.go

```diff
@@ -99,20 +112,15 @@ func enableShell() error {
 		return err
 	}
 
+	log.Info("Attempted to enable the shell for root")
```

Comment:
I think in needs a better log message, something like: Shell for the root user has been enabled.
attempted - doesn't mean success.